### DISCUSSION TITLE
share a calendar

### DISCUSSION THREAD

**Discussion by hppyworld** [ckulka/baikal-docker#60] [Q&A]
How to share a calendar. where can i make configs? 
Thanks for help

**Comment by ckulka** [comment]
Hi @hppyworld ,

this looks like a question related to "How to use Baikal" rather than something specific to the Baikal Docker image itself, so I'd defer you to the Baikal site <https://sabre.io/baikal/> and their repository <https://github.com/sabre-io/Baikal>.

Is this correct, or did you mean something related to setting up the container, e.g. how to make it accessible from somewhere after you already configured Baikal?

**Comment by hppyworld** [comment]
Actually How to share a calendar with docker image. Because as i see there are some extra things needs be configured as its not coming by default in web ui. This configuration needs to be done in image or container. 
thanks for help.
